manifests/init.pp

```diff
@@ -68,24 +68,30 @@
   } ->
 
   # remove default web site
-  iis::manage_site {'Default Web Site':
-    ensure    => absent,
-    site_path => 'any',
-    app_pool  => 'DefaultAppPool',
-    require   => Windowsfeature['Web-WebServer'],
+  iis_site {'Default Web Site':
+    ensure           => absent,
+    applicationpool => 'DefaultAppPool',
+    require          => Windowsfeature['Web-WebServer'],
   } ->
 
   # application in iis
-  iis::manage_app_pool { "${_chocolatey_server_app_pool_name}":
-    enable_32_bit           => true,
-    managed_runtime_version => 'v4.0',
+  iis_application_pool { "${_chocolatey_server_app_pool_name}":
+    ensure                    => 'present',
+    state                     => 'started',
+    enable32_bit_app_on_win64 => true,
+    managed_runtime_version   => 'v4.0',
   } ->
-  iis::manage_site {'chocolatey.server':
-    site_path  => $_chocolatey_server_location,
-    port       => "${_chocolatey_server_app_port}",
-    ip_address => '*',
-    app_pool   => "${_chocolatey_server_app_pool_name}",
-    require    => Package['chocolatey.server'],
+  iis_site {'chocolateyserver':
+    ensure          => 'started',
+    physicalpath    => $_chocolatey_server_location,
+    applicationpool => "${_chocolatey_server_app_pool_name}",
+    bindings        =>  [
+      {
+        'bindinginformation' => '*:80:',
+        'protocol'           => 'http'
+      }
```

Comment:
Are bindings actually required or is there a default?

Reply:
@jpogran ^^ ?

Reply:
Not required, default is port 80. Beware not getting ride of the 'Default Web Site' if you want port 80, it starts first and takes the port causing your site to fail

Reply:
@jpogran just removed the bindings on the last PR